CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 - Now all `Element` have a default length of `torch.zeros((1))`, fixing occasional issues with using elements without length, such as `Marker`, `BPM`, `Screen`, and `Aperture`. (see #143) (@cr-xu)
 - Fix bug in `Cavity` `_track_beam` (see [#150](https://github.com/desy-ml/cheetah/issues/150)) (@jp-ga)
+- Fix bugs when tracking elements with `tilt` or `misalignment`. (see #157) (@cr-xu)
```

Comment:
@cr-xu Did you fix anything other than the vectorisation? I guess because the changelog is with respect to the previous release (and not master), rather than adding a bug fix, we should add the issue and your name to the breaking vectorisation change entry. Does that make sense?

Reply:
No, that's only with the vectorisation. Yes it makes sense to move it in the vectorisation entry.